state/store.go

```diff
@@ -525,16 +527,20 @@ func (store dbStore) saveValidatorsInfo(height, lastHeightChanged int64, valSet
 
 // ConsensusParamsInfo represents the latest consensus params, or the last height it changed
 
-// LoadConsensusParams loads the ConsensusParams for a given height.
-func (store dbStore) LoadConsensusParams(height int64) (tmproto.ConsensusParams, error) {
-	empty := tmproto.ConsensusParams{}
+// Allocate empty Consensus params at compile time to avoid multiple allocations during runtime
+var (
+	empty   = types.ConsensusParams{}
+	emptypb = tmproto.ConsensusParams{}
+)
```

Comment:
Is this copped from somewhere else? It seems dubious that this would actually save any measurable memory or time, and just opens us up to weird testing flukes. If there's a good way to omit this pattern, I would.

Reply:
Yeah it's copied from the original PR

Reply:
Agree with Sam. This is saving us almost nothing. As far as I know, global variables are not allocated at compile time. I'm not really sure I know what that means. They're allocated when the program starts. This just means they're allocated only once which really saves us so little here.